admin/assets/javascripts/global-scripts.js

```diff
@@ -3,190 +3,208 @@
+        }
+        radioEl.attr('data-checked', null)
+        if (radioEl.is(':checked')) {
+          $($(this)).attr('data-checked', true)
+          stickyBanner.toggle(validationStatus)
+        }
+      })
     }
   }
 
-  if ($('#checkFormsList').length > 0) tickAllCheckboxes('#checkFormsList')
-  if ($('#pupilsList').length > 0) tickAllCheckboxes('#pupilsList')
-  if ($('#attendanceList').length > 0)tickAllCheckboxes('#attendanceList')
-
-  if ($('#checkFormsList').length > 0) disableCheckAll('#checkFormsList')
-  if ($('#pupilsList').length > 0) disableCheckAll('#pupilsList')
-  if ($('#attendanceList').length > 0) disableCheckAll('#attendanceList')
-
   /**
-   * Pupils not taking the check methods.
+   * Enable/disable confirmation button from sticky banner.
+   * @type {{toggle: toggle, countCheckboxes: countCheckboxes}}
    */
-  var pupilsNotTakingCheck = function () {
+  var stickyBanner = {
     /**
-     * Enable/disable confirmation button from sticky banner.
      * @param status
      */
-    var enableButtonStatus = function (status) {
+    toggle: function (status) {
       if (status === true) {
+        stickyBanner.countCheckboxes()
         $('#stickyBanner').addClass('show')
       } else {
         $('#stickyBanner').removeClass('show')
       }
-    }
-
+    },
     /**
-     * Is radio button checked?
+     * @param checkboxParent
      */
-    var isRadioChecked = function () {
-      var el = $('input:radio[name="attendanceCode"]:checked')
-      if (el.length > 0) {
-        enableButtonStatus(true)
-      } else {
-        enableButtonStatus(false)
-      }
+    countCheckboxes: function (checkboxParent) {
+      var el = $((checkboxParent || '.multiple-choice-mtc') + ' > input:checkbox:checked')
+      $('#totalPupilsSelected').text(el.length)
     }
+  }
 
+  /**
+   * Methods for 'pupils not taking the check'.
+   * @type {{isRadioChecked: isRadioChecked, isCheckboxChecked: isCheckboxChecked, validateForm: validateForm}}
+   */
+  var pupilsNotTakingCheck = {
     /**
-     * Count checkboxes, populate sticky <span>.
+     * Is radio button checked?
+     * @returns {boolean}
      */
-    var countCheckboxes = function () {
-      var el = $('.multiple-choice-mtc > input:checkbox:checked')
-      $('#totalPupilsSelected').text(el.length)
-    }
-
+    isRadioChecked: function () {
+      var el = $('input:radio[name="attendanceCode"]:checked')
+      return (el.length > 0)
+    },
     /**
      * Is there at least one checkbox checked?
+     * @returns {boolean}
      */
-    var isCheckboxChecked = function () {
+    isCheckboxChecked: function () {
       var el = $('.multiple-choice-mtc > input:checkbox:checked')
-      if (el.length > 0) {
-        enableButtonStatus(true)
-      } else {
-        enableButtonStatus(false)
-      }
+      return (el.length > 0)
+    },
+    validateForm: function () {
+      return (pupilsNotTakingCheck.isRadioChecked() && pupilsNotTakingCheck.isCheckboxChecked())
```

Comment:
dont need the brackets

Reply:
@GeorgeChatzigiannis I'm aware of alternative formats, I find the brackets useful to make the code more readable, especially with a two character indentation.

Reply:
@GeorgeChatzigiannis I'll remove the parenthesis just for the sake of get this PR approved TODAY. Thanks.